includes/polling/mempools/fortimail.inc.php

```diff
@@ -0,0 +1,8 @@
+<?php
+echo 'FortiMail MemPool';
+$mempool['perc'] = snmp_get($device, 'FORTINET-FORTIMAIL-MIB::fmlSysMemUsage.0', '-OvQ');
+// There is no OID for MemCapacity. Using fixed value based on "diag hardware sysinfo"
+$mempool['total'] = 4123984 * $mempool['mempool_precision'];
```

Comment:
Could you run a complete SNMPWALK of your device, and search inside it if this value appears somewhere. Would be a little bit annoying to hardcode this especially when next hardware revision will change the embedded memory ...

Reply:
Thanks for your input, but unfortunatly this value is really unavailable using SNMP. I've already opened a support ticket at Fortinet regarding that.

Reply:
I just received feedback from Fortinet that they won't implement this OID any time soon.